modules/utils.py

```diff
@@ -48,4 +48,7 @@ def keywords_without_word(language, word):
 	new_keys = subscribe_keywords(language)
 	if word in new_keys:
 		new_keys.remove(word)
-	return new_keys
\ No newline at end of file
+	return new_keys
+
+def is_not_ascii(message):
+	return any(ord(char) >= 128 for char in message)
```

Comment:
Isn't this more like `has_ascii` or `is_ascii`? It returns `True` when ascii is present.

Reply:
I think I'm just confused what "ascii" refers to. I guess >= 128 is unicode?

Reply:
Yes. Over 128 is unicode. I considered just reversing this function (so it runs all(), since any and all seem to be structurally similar with regard to speed in the docs, and returns True if ASCII) but decided against since that would have introduced a negative if statement into the conditional where this is used.